docs/pages/web/slimbanner.js

```diff
@@ -589,7 +615,11 @@ Tooltip provides helpful information regarding an interactive UI element, typica
 }
 
 export async function getStaticProps(): Promise<{| props: {| generatedDocGen: DocGen |} |}> {
+  const docGen = await docgen({ componentName: 'SlimBanner' });
+
+  docGen.props.message.flowType.raw = '{| string | React.Element<typeof Text>>> |}';
```

Comment:
This is the logic that should be in GeneratedPropTable. We want to handle this case universally, not as a one-off

Reply:
I'll fix that in a separate PR. OOS for this PR